global-news-app/src/assets/styles/types.ts

```diff
@@ -1,9 +1,3 @@
-export enum buttonVariants {
-  PRIMARY = "primary",
-  SECONDARY = "secondary",
-  TEXT = "text",
-}
-
 export type Spacing = {
```

Comment:
you can you `Record` utility type for this: https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkeys-type

for example: 
`type Spacing = Record<number, string>;`

or more than that, you can use `Record` with a union of specific keys if you know them beforehand
```
type NumericKeys = 0 | 1 | 2 | 3; 
type Spacing = Record<NumericKeys, string>;
```